src/Resources/config/core.php

```diff
@@ -136,9 +137,12 @@
             ->tag('jms_translation.extractor', [
                 'alias' => 'sonata_admin',
             ])
-            ->deprecate(sprintf(
-                'The service "%%service_id%%" is deprecated since sonata-project/admin-bundle 3.72 and will be removed in 4.0. Use "%s" service instead.',
-                Sonata\AdminBundle\Translator\Extractor\AdminExtractor::class
+            ->deprecate(...BCDeprecationParameters::forConfig(
+                sprintf(
+                    'The service "%%service_id%%" is deprecated since sonata-project/admin-bundle 3.72 and will be removed in 4.0. Use "%s" service instead.',
+                    Sonata\AdminBundle\Translator\Extractor\AdminExtractor::class
+                ),
+                '3.72'
             ))
```

Comment:
```suggestion
            ->deprecate(...BCDeprecationParameters::forConfig(sprintf(
                'The service "%%service_id%%" is deprecated since sonata-project/admin-bundle 3.72 and will be removed in 4.0. Use "%s" service instead.',
                Sonata\AdminBundle\Translator\Extractor\AdminExtractor::class
            ), '3.72'))
```